coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/BusinessLogic/RegistrationBusinessLogic.cs

```diff
@@ -153,9 +154,23 @@ public async Task CreateDocument(IFormFile document, string userName)
                     hash = builder.ToString();
                 }
             }
-            await _dbAccess.UploadDocument(name,documentContent,hash,userName);
+            await _dbAccess.UploadDocument(name, documentContent, hash, userName);
```

Comment:
Cleaning up is in general good but I would not commend to do that in general but if you remove or add spaces in parts which don't have anything to do with your implementation, please do that in a separate commit, at least in the future